### ISSUE TITLE
Make files available from S3 via https

### ISSUE THREAD

**Issue by yarikoptic** [OpenNeuroOrg/openneuro#1400]
For a variety of reasons it would be great to get data available from S3 via https, but unfortunately ATM it doesn't work:

```shell
$> wget https://openneuro.org.s3.amazonaws.com/ds000001/sub-01/anat/sub-01_T1w.nii.gz
--2019-11-25 11:44:47--  https://openneuro.org.s3.amazonaws.com/ds000001/sub-01/anat/sub-01_T1w.nii.gz
Resolving openneuro.org.s3.amazonaws.com (openneuro.org.s3.amazonaws.com)... 52.217.45.60
Connecting to openneuro.org.s3.amazonaws.com (openneuro.org.s3.amazonaws.com)|52.217.45.60|:443... connected.
The certificate's owner does not match hostname ‘openneuro.org.s3.amazonaws.com’
```

**Comment by effigies** [comment]
The certificate is for `*.s3.amazonaws.com`, so apparently having a `.` in our bucket name breaks this.

**Comment by nellh** [comment]
Went ahead and fixed this in the remote configuration in #1700, this way as we apply fixes to the remote configuration datasets will start using HTTPS S3 publicurl and new datasets will do so as soon as that's deployed.
